_saas-integrations/adroll/v1/adroll-v1.md

```diff
@@ -0,0 +1,115 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Adroll (v1)
+permalink: /integrations/saas/adroll
+keywords: adroll, integration, schema, etl adroll, adroll etl, adroll schema
+layout: singer
+# input: false
+
+key: "adroll-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "adroll"
+display_name: "Adroll"
```

Comment:
The company actually has their name as `AdRoll`, with a capitalized `R`. It's nitpicky, but our docs should be consistent with however the company spells their name. This will need to be updated here and anywhere else you have `Adroll`, but `adroll` can stay as-is.